tutorial_sarscov2.md

```diff
@@ -125,106 +149,165 @@ aligner = bwa
 
 Check what will be executed:
 ```bash
-snakemake -s vpipe.snake --dryrun -p --cores 2
+./vpipe --dryrun -p --cores 2
 ```
 
 As it is your first run of V-pipe, this will also generate the sample collection table.
 Check `samples.tsv` in your editor.
 
-Note that the demo files you downloaded have only have reads of length 150.
-V-pipe's defaults parameters are optimized for reads length of 200 ; add the third column in the file:
+Note that the demo files you downloaded only have reads of length 150.
+V-pipe's default parameters are optimized for reads of length 250 ; add the third column in the file:
 
 ```
 SRR10903401	20200102	150
 SRR10903402	20200102	150
 ```
 
 > **Tips:** Always check the content of the `samples.tsv` file.
->  - If you didn't use the [correct structure](#preparing-a-small-dataset) this file might end up empty or some entries might be missing.
->  - You can safely delete it and re-run the `--dryrun` to regenerate it
+>  - If you didn't use the [correct structure](#preparing-a-small-dataset), this file might end up empty or some entries might be missing.
+>  - You can safely delete it and re-run the `--dryrun` to regenerate it.
 
 Run the V-pipe analysis (the necessary dependencies will be downloaded and installed in conda environments managed by snakemake):
 
 ```bash
-snakemake -s vpipe.snake --use-conda -p --cores 2
+./vpipe --use-conda -p --cores 2
 ```
 
+## Output
 
-## Larger dataset
+The Wiki contains an overview of the [output files](https://github.com/cbg-ethz/V-pipe/wiki/output).
+The output of the SNVs calling is aggregated in a standard [VCF](https://en.wikipedia.org/wiki/Variant_Call_Format) file, located in
+`samples/`*\{hierarchy\}*`/variants/SNVs/snvs.vcf`, you can open it with your favorite VCF tools for aggregated or downstream processing.
 
-Log onto [Euler](https://scicomp.ethz.ch/wiki/Euler) with ETH account.
+*[VCF]: Variant Call Format
 
-```bash
-ssh login.euler.ethz.ch
-```
+There is also a visual report generated in `samples/`*\{hierarchy\}*`/visualization/index.html` that you can simply open with your favorite web browser (Firefox, Chrome). You will get an output similar to that:
 
-There is a much larger test data that you can use:
+![SARS-CoV-2 report visualisation]({{ "/img/visualisation-sars-cov2.png" | relative_url }})
 
-```bash
-ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/
-ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/PRJNA6*
-ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/PRJNA6*/sample*/
+You can use the mouse selection to zoom in region and double clicks to zoom out.
+
+### Expected output
+
+The small dataset that we used in this first part has been analyzed by [doi:10.1093/nsr/nwaa036](https://doi.org/10.1093/nsr/nwaa036).
```

Comment:
'in this first part' is confusing.
The results of the original analysis (using bwa, samtools mpileup, and bcftools) are displayed in Table 2 in the article ...